views/Home/Home.jsx

```diff
@@ -108,6 +111,19 @@ export default class HomeView extends React.Component {
     this.props.navigation.navigate('Feed', { channel, loggedInUser: user });
   }
 
+  onLongPressChannel = (channelId, channelName) => {
+    const { channels, user } = this.state;
+    var channel;
+
+    if (['all', 'subs', 'myPosts'].includes(channelId)) channel = { _id: channelId, name: channelName };
+    else channel = _.head(_.filter(channels, { _id: channelId }));
```

Comment:
This can be done without lodash using es6 syntax (I don't really like that we have lodash as a dependency)
```
channel = channels.find(channel => channel._id === channelId)
```